.circleci/build-all.sh

```diff
@@ -5,6 +5,11 @@ cd "$DIR"
 
 if [[ -n "${CIRCLECI}" ]]; then
   echo "Docker logs are located in the CircleCI build artifacts"
+  if [ "${DOCKER_USER}" == "" ] || [ "${DOCKER_PASS}" == "" ]; then
+      echo "Skipping Login to Dockerhub, credentials not available."
+  else
+      echo "${DOCKER_PASS}" | docker login -u="${DOCKER_USER}" --password-stdin
+  fi
```

Comment:
I don't think we need to login for the build scripts, just deploy

Reply:
It's my understanding that *any* docker actions (pull/push) will be subject to the rate-limiting, so any images used during build would count against that as well.

Reply:
build doesn't do any docker actions